app/src/main/java/org/simple/clinic/widgets/qrcodescanner/QrCodeScannerView.kt

```diff
@@ -47,9 +53,18 @@ constructor(
 
     val preview = Preview.Builder().build()
 
+    val analyzer = ImageAnalysis.Builder()
+        .setBackpressureStrategy(ImageAnalysis.STRATEGY_KEEP_ONLY_LATEST)
+        .build()
+        .also {
```

Comment:
This can be an `.apply` block, right?

Reply:
We can use `.apply` as well, but since it's not a direct object configuration I personally prefer `.also` . Because we are taking the build `ImageAnalysis` as argument and using it to set the analyzer. Should I change this to `.apply`?

Reply:
It's up to you. If you want to leave it as is, that's fine as well.